script/cadence/contracts/FlowColdStorageProxy.cdc

```diff
@@ -1,5 +1,5 @@
-import FlowToken from 0x7e60df042a9c0868
-import FungibleToken from 0x9a0766d93b6608b7
+import FlowToken from 0x0ae53cb6e3f42a79
+import FungibleToken from 0xee82856bf20e2aa6
```

Comment:
is it mainnet addr or testnet one holding the contract?

Reply:
It's emulator addresses. When deploying to any other network use the proper addresses.